homeassistant/components/sfr_box/__init__.py

```diff
@@ -65,9 +65,10 @@ async def async_setup_entry(hass: HomeAssistant, entry: SFRConfigEntry) -> bool:
     device_registry = dr.async_get(hass)
     device_registry.async_get_or_create(
         config_entry_id=entry.entry_id,
+        connections={(dr.CONNECTION_NETWORK_MAC, system_info.mac_addr)},
         identifiers={(DOMAIN, system_info.mac_addr)},
         name="SFR Box",
-        model=system_info.product_id,
+        model=None,
```

Comment:
`model` and `model_id` are currently set to the same value.

Forced to `None` to ensure existing `model` value is removed.